EIPS/eip-1328.md

```diff
@@ -1,21 +1,18 @@
 ---
 eip: 1328
 title: WalletConnect Standard URI Format
-author: ligi <ligi@ligi.de>, Pedro Gomes <pedrouid@protonmail.com>
+author: ligi <@ligi>, Pedro Gomes <@pedrouid>
 type: Standards Track
 category: ERC
-status: Stagnant
+status: Draft
 created: 2018-08-15
 discussions-to: https://ethereum-magicians.org/t/wallet-connect-eip/850
+description: Standard for WalletConnect URIs
 ---
 
-## Simple Summary
-
-A standard to create WalletConnect URIs to initiate connections between applications and wallets.
-
 ## Abstract
 
-This standard defines how the data to connect some application and a wallet can be encoded with a URI. This URI can then be shown either as a QR code or for mobile to mobile as a link.
+This standard defines how the data to connect some application and a wallet can be encoded with a URI. This URI can then be shown either as a QR code or as a link.
```

Comment:
@pedrouid do we perhaps want to remove "This URI can then be shown either as a QR code or as a link." - think it adds no value and restricts us without a need. e.g. how does the "copy" fall in place there.

Reply:
I feel like this is already addressed when "link" is mentioned since it's a common pattern to include `copy to clipboard` for website links as well